pandas/core/frame.py

```diff
@@ -2915,6 +2915,59 @@ def rename(self, index=None, columns=None, **kwargs):
     @Appender(_shared_docs['fillna'] % _shared_doc_kwargs)
     def fillna(self, value=None, method=None, axis=None, inplace=False,
                limit=None, downcast=None, **kwargs):
+        """
+        Examples
```

Comment:
Putting those examples here won't work, as now the rest of the docstring is appended to the examples, while the examples should come at the end.